pkgs/applications/office/tagainijisho/default.nix

```diff
@@ -1,16 +1,45 @@
-{ lib, stdenv, fetchurl, qt4, cmake, sqlite }:
-
-stdenv.mkDerivation rec {
+{ lib, mkDerivation, fetchFromGitHub, fetchurl, qtbase, qttools, cmake, sqlite }:
+let
+  jmdict = fetchurl {
+    urls = [
+      "http://ftp.edrdg.org/pub/Nihongo/JMdict.gz"
+      "http://www.bcit-broadcast.com/monash/JMdict.gz"
+      "ftp://ftp.net.usf.edu/pub/ftp.monash.edu.au/pub/nihongo/JMdict.gz"
+    ];
+    hash = "sha256-oreu0I0+MEQftEYSK2CqrxaIXHzRgyjJZ5+ik43SShw=";
+  };
+  kanjidic2 = fetchurl {
+    url = "https://www.edrdg.org/kanjidic/kanjidic2.xml.gz";
+    hash = "sha256-uW53CemginkX78Lt5T9BuUTQVWLkqVGnyA3RlJyP458=";
+  };
+  kanjivg = fetchurl {
+    url = "https://github.com/KanjiVG/kanjivg/releases/download/r20160426/kanjivg-20160426.xml.gz";
+    hash = "sha256-tApHS+CTZphYajFgFDFOWNJWhahZUBdfhuXYQ6b+Xsc=";
+  };
+in mkDerivation rec {
   pname = "tagainijisho";
-  version = "1.0.3";
+  version = "1.2.0";
 
-  src = fetchurl {
-    url = "https://github.com/Gnurou/tagainijisho/releases/download/${version}/tagainijisho-${version}.tar.gz";
-    sha256 = "0kmg1940yiqfm4vpifyj680283ids4nsij9s750nrshwxiwwbqvg";
+  src = fetchFromGitHub {
+    owner = "Gnurou";
+    repo = pname;
+    rev = version;
+    hash = "sha256-EDGWpJ80xQ9436RPK7tQ3iY3eE1NfSuxEt40bZlMtoc=";
```

Comment:
Is there anything wrong with the released artifact at <https://github.com/Gnurou/tagainijisho/releases/download/1.2.0/tagainijisho-1.2.0.tar.gz>? It seems to already contain the third-party dictionaries.

Reply:
seems to work just fine, don't remember why I initially switched away from that, tbh